integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala

```diff
@@ -316,4 +324,17 @@ object IndexServer extends ServerInterface {
       Array(new Service("security.indexserver.protocol.acl", classOf[ServerInterface]))
     }
   }
+
+  def startAgingFolders(): Unit = {
+    val runnable = new Runnable() {
+      def run() {
+        val age = System.currentTimeMillis() - agePeriod.toLong
+        CarbonUtil.agingTempFolderForIndexServer(age)
+        LOGGER.info(s"Complete age temp folder ${CarbonUtil.getIndexServerTempPath}")
+      }
+    }
+    val ags: ScheduledExecutorService = Executors.newSingleThreadScheduledExecutor
+    ags.scheduleAtFixedRate(runnable, 1000, 3600000, TimeUnit.MICROSECONDS)
+    LOGGER.info("index server temp folders aging thread start")
```

Comment:
Please add this log inside the thread so that we can know each time the deletion thread was executed

Reply:
under run func there already has logs.
      def run() {
        val age = System.currentTimeMillis() - agePeriod.toLong
        CarbonUtil.agingTempFolderForIndexServer(age)
        LOGGER.info(s"Complete age temp folder ${CarbonUtil.getIndexServerTempPath}")
      }